deploy/1.8+/metrics-server-deployment.yaml

```diff
@@ -40,6 +40,7 @@ spec:
         securityContext:
```

Comment:
I think we can align with helm chart defaults on securityContext, cause it works and adds more options.

https://github.com/helm/charts/blob/master/stable/metrics-server/values.yaml#L81